src/molecules/formfields/ToggleField/index.js

```diff
@@ -9,30 +9,32 @@ import Layout from 'atoms/Layout';
 
 import styles from '../shared/formfields.module.scss';
 
-function renderChoices(choices, input) {
+const renderChoices = (choices, input) => {
   if (!choices) return null;
```

Comment:
Although, it may not make a difference here. I find that it's easier to reason with code flow with the use of guard clauses over conditionals outside of the function.